solace_prometheus_exporter.go

```diff
@@ -2814,11 +2891,36 @@ func main() {
 		"private-key",
 		"If using TLS, you must provide a valid private key in PEM format. Can be set via config file, cli parameter or env variable.",
 	).ExistingFile()
+	hashPwd := kingpin.Flag(
+		"hash-password",
+		"To use basic auth, you must provide hashed passwords. Use this flag to create a hashed password.",
+	).String()
+	enableBasicAuth := kingpin.Flag(
```

Comment:
"enable-basic-auth" feels a little redundant.
Why not activate authentication as soo as "users-file" was given.

Reply:
Good point - I thought about that as well. 
However for TLS there is a similar flag and one could argue that setting a cert and key file could be used to enable TLS instead of having a separate "enable" flag. 
For me both would be fine as long as it is kinda consistent.

Reply:
Ok i am with you. For TLS "--enable-tls" makes totally sense.
And doing it consistent is also reasonable. 

Leave it as it is.